github/prci_github/internals.py

```diff
@@ -503,20 +517,18 @@ def take_tasks(self):
         tasks = []
         for pull in PullRequests(self.repo):
             tasks_done_per_pr[pull.pull.number] = 0
-            for task in pull.tasks(self.tasks_config_path, self.job_cls):
+
+            pr_tasks = pull.tasks(self.tasks_config_path, self.job_cls)
+            logging.debug('Tasks in the PR: %s', [t.name for t in pr_tasks])
+            logging.debug('Tasks done PR: %s', tasks_done_per_pr)
```

Comment:
I find logging this a bit odd. On each PR, it prints the number of tasks done for all previous PRs and always 0 for the current one, regardless of how many tasks are done. We should either drop this, log an accurate singel number for current PR or log it at the end of the loop for all PRs.

Reply:
Yeah, from my PVO is pretty much useless at this point. I forgot to remove that from my debugging.